features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx

```diff
@@ -169,77 +182,68 @@ function TryOutController(props) {
-            }
-        }).catch((error) => {
-            if (process.env.NODE_ENV !== 'production') {
-                console.error(error);
-            }
-            const { status } = error;
-            if (status === 404) {
-                setNotFound(true);
-            }
-        });
-        const promisedKeyManagers = restApi.getKeyManagers();
-        promisedKeyManagers
-            .then((response) => {
-                const responseKeyManagerList = [];
-                response.body.list.map((item) => responseKeyManagerList.push(item));
-                setKeyManagers(responseKeyManagerList);
-            })
-            .catch((error) => {
+            }).catch((error) => {
                 if (process.env.NODE_ENV !== 'production') {
-                    console.log(error);
+                    console.error(error);
                 }
                 const { status } = error;
                 if (status === 404) {
                     setNotFound(true);
                 }
             });
+            const promisedKeyManagers = restApi.getKeyManagers();
+            promisedKeyManagers
+                .then((response) => {
+                    const responseKeyManagerList = [];
+                    response.body.list.map((item) => responseKeyManagerList.push(item));
+                    setKeyManagers(responseKeyManagerList);
+                })
+                .catch((error) => {
+                    if (process.env.NODE_ENV !== 'production') {
+                        console.log(error);
+                    }
+                    const { status } = error;
+                    if (status === 404) {
+                        setNotFound(true);
+                    }
+                });
+        }
     }, []);
 
 
     /**
      * Generate access token
      * */
     function generateAccessToken() {
-        setIsUpdating(true);
-        const applicationPromise = Application.get(selectedApplication);
-        applicationPromise
-            .then((application) => application.generateToken(
-                selectedKeyManager,
-                selectedKeyType,
-                3600,
-                scopes,
-            ))
-            .then((response) => {
-                console.log('token generated successfully ' + response);
-                setShowToken(false);
-                if (selectedKeyType === 'PRODUCTION') {
-                    setProductionAccessToken(response.accessToken);
-                } else {
-                    setSandboxAccessToken(response.accessToken);
-                }
-                setIsUpdating(false);
-            })
-            .catch((error) => {
-                console.error(error);
-                const { status } = error;
-                if (status === 404) {
-                    setNotFound(true);
-                }
-                setIsUpdating(false);
-            });
+        if (!api.lifeCycleStatus && api.lifeCycleStatus.toLowerCase() === 'prototyped') {
```

Comment:
shouldn't the subscription call happen if api.lifeCycleStatus.toLowerCase() !== 'prototyped') ?

Reply:
fixed issue.